client/components/card-heading/index.jsx

```diff
@@ -9,7 +9,7 @@ import './style.scss';
 //Sizes 47, 21, and 11 are deprecated; use the nearest equivalent
 const validTypeSizes = [ 54, 48, 47, 36, 32, 24, 21, 20, 16, 14, 12, 11 ];
 
-function CardHeading( { tagName = 'h1', size = 20, isBold = false, className, children } ) {
+function CardHeading( { tagName = 'h1', size = 20, isBold = false, className, id, children } ) {
```

Comment:
Can you update `README.md` for this component, too?

Reply:
Good catch, I've added a new prop to the readme.